molgenis-security/src/test/java/org/molgenis/security/account/AccountControllerTest.java

```diff
@@ -120,69 +118,6 @@ public void activateUserMolgenisUserException() throws Exception {
     verify(accountService).activateUser("123");
   }
 
-  @Test
-  public void registerUser_activationModeUserProxy() throws Exception {
-    when(authenticationSettings.getSignUp()).thenReturn(true);
-    when(authenticationSettings.getSignUpModeration()).thenReturn(false);
-
-    this.mockMvc
-        .perform(
-            post("/account/register")
-                .header("X-Forwarded-Host", "website.com")
-                .param("username", "admin")
-                .param("password", "adminpw-invalid")
-                .param("confirmPassword", "adminpw-invalid")
-                .param("email", "admin@molgenis.org")
-                .param("lastname", "min")
-                .param("firstname", "ad")
-                .param("recaptch", "validCaptcha")
-                .contentType(MediaType.APPLICATION_FORM_URLENCODED))
-        .andExpect(status().isOk())
-        .andExpect(
-            content()
-                .string(
-                    "{\"message\":\""
-                        + AccountController.REGISTRATION_SUCCESS_MESSAGE_USER
-                        + "\"}"));
-    ArgumentCaptor<User> molgenisUserCaptor = ArgumentCaptor.forClass(User.class);
-    ArgumentCaptor<String> baseActivationUriCaptor = ArgumentCaptor.forClass(String.class);
-    verify(accountService)
-        .createUser(molgenisUserCaptor.capture(), baseActivationUriCaptor.capture());
-    assertEquals(baseActivationUriCaptor.getValue(), "http://website.com/account/activate");
```

Comment:
I think we should test whether the URLs that we generate are correct

Reply:
Tested in the API test